tests/requirements.txt

```diff
@@ -0,0 +1,10 @@
+# test libraries
+coverage
+djangocms_helper
+flake8
+isort
+factory-boy
+djangocms-column
+https://github.com/divio/djangocms-moderation/archive/release/1.0.x.zip
+https://github.com/divio/djangocms-versioning/archive/master.zip
+https://github.com/divio/djangocms-text-ckeditor/archive/support/4.0.x.zip
```

Comment:
no internal search/version locking, is it because optional?

Reply:
I've only installed stuff that is required for data insertion to work